### DISCUSSION TITLE
Does Rectangle support 1/4 & 3/4 window sizes?

### DISCUSSION THREAD

**Discussion by zakiaziz** [rxhanson/Rectangle#480] [Q&A] [answered]
It'd be great if rectangle can support that. I constantly see my self resorting to those ratios for my IDE/terminal. Please let me know if I'm over looking something.

**Comment by rxhanson** [comment, marked as answer]
I think you might have just missed the fourths section. Expand the triangle & ellipsis in the prefs window and look on the bottom left.

**Reply by zakiaziz** [reply]
You're totally right. Completely missed it. Guess it was difficult to see in dark mode on my monitor. Thank you!

But still not possible to do 3/4's correct?

**Reply by rxhanson** [reply]
Execute 2nd or 3rd Fourth twice.
